fhir-term-graph/src/main/kubernetes/elasticsearch.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
```

Comment:
I'm not sure what having the kubernetes artifacts in the repo does for us.
For the rest of the project, we stop at producing the Docker images and Kubernetes-related work as been pushed off to `Alvearie/alvearie-helm` or `Alvearie/health-patterns` ...would these make more sense over there?

Reply:
comment applies to the whole fhir-term-graph/src/main/kubernetes directory

Reply:
just test artifacts ... it may make sense to move to src/test/

Reply:
yeah, i think i'd be more comfortable with them under src/test (although I still don't think we need them here at all)

Reply:
I'll move them to src/test. I included them as I felt they were useful for reference. I can move them to offline documentation if you'd prefer.